src/hooks/useConsent.ts

```diff
@@ -104,6 +113,36 @@ export const useConsent = () => {
     useConsentDirectives,
     useUpdateProjectConsentDirective,
     useShouldRenderConsentScreen,
+  } as {
```

Comment:
Why did we need this cast?

Reply:
See https://lifeomic.slack.com/archives/C05KMKQNTAR/p1707948259517759 or the failed earlier builds. Happy to fix some other way but that is what was decided there.

Reply:
We are not building the FL member app as an SDK is probably the biggest difference.